spec/factories/purpose_factories.rb

```diff
@@ -139,4 +139,14 @@
     sequence(:name) { |n| "Child tube purpose #{n}" }
     target_type { 'Tube' }
   end
+
+  factory :tube_rack_purpose_48, class: TubeRack::Purpose do
```

Comment:
The factory is named with '48' but has both sizes?

Reply:
Yeah I wasn't quite sure how to structure it, what would you do?

Reply:
maybe the default 'tube_rack_purpose' factory is size 96, and we just override size and name if we want something else...

Reply:
Sure, I'll do that.